run_wizard.py

```diff
@@ -845,7 +845,8 @@ def _validate_input(self, user_value, default=None, choices=None,
             if value is None:
                 self._print("\nValue is not a valid hostname\n")
         elif datatype == "version":
-            allowed = re.compile("^[\d][.][\d][.]([A-Z\d]+)$", re.IGNORECASE)
+            allowed = re.compile("^[0-9]+[.][0-9]+[.]([A-Z\d]+)$", re.IGNORECASE)
```

Comment:
You should be able to still use the [\d] in the regex.

Reply:
done